File: android-aop-annotation/src/main/java/com/flyjingfish/android_aop_annotation/anno/AndroidAopMethod.java
package com.flyjingfish.android_aop_annotation.anno;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.METHOD)
public @interface AndroidAopMethod {
    String value();
    String pointCutClassName();
}

File: android-aop-annotation/src/main/java/com/flyjingfish/android_aop_annotation/anno/AndroidAopMethod.kt
package com.flyjingfish.android_aop_annotation.anno



/**
 * 这个注解使用本库者用不到
 */
@Retention(AnnotationRetention.BINARY)
@Target(AnnotationTarget.FUNCTION)
annotation class AndroidAopMethod(val value: String, val pointCutClassName: String)
